kanopi/patternlab-hjf
# tw23946677-ie11-icons - replacing mask-image with background-image

## Description
This PR replaces mask-image styles with background-image styles for ie11 coverage.

## Tasks:
https://kanopi.teamwork.com/#/tasks/23946684 (menu arrows)
https://kanopi.teamwork.com/#/tasks/23946677 (social icons)

## Validation Instructions
1. go to homepage in ie11
1. Validated social icons in the footer render per the designs (https://app.zeplin.io/project/5c912e0c7af93805d083b62f/screen/5c950f1d96545605b2738822)
1. Validate arrow icons in main menu render correctly in ie11
1. Check social icons and menu arrows in at least one Mac browser to ensure they still look correct
